gruntfile.js

```diff
@@ -1,4 +1,39 @@
 module.exports = function (grunt) {
+
+    const aiCoreDefaultNameReplacements = [
+       {
+           src: "./src/JavaScriptSDK.Enums/SdkCoreNames.ts",
+           import: "./src/JavaScriptSDK.Enums/SdkCoreNames"
+       },
+       {
+           src: "./src/JavaScriptSDK.Enums/ConfigEnums.ts",
+           import: "./src/JavaScriptSDK.Enums/ConfigEnums"
+       }
+   ];
+
+   const aiDefaultNameReplacements = [
+        {
+            src: "../shared/AppInsightsCore/src/JavaScriptSDK.Enums/SdkCoreNames.ts",
+            import: "@microsoft/applicationinsights-core-js"
+        },
+        {
+            src: "../shared/AppInsightsCore/src/JavaScriptSDK.Enums/ConfigEnums.ts",
+            import: "@microsoft/applicationinsights-core-js"
+        },
+        {
+            src: "../../shared/AppInsightsCore/src/JavaScriptSDK.Enums/SdkCoreNames.ts",
```

Comment:
The different "depths" of the references are due to the project layout so

`./AISKU` only needs to use `../shared/...` to get to the Core source code while
`./extensions/xxxx` needs to use `../../shared/...` to get there.